org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/relaxng/grammar/rng/RNGDocumentLinkingExtensionsTest.java

```diff
@@ -103,4 +103,21 @@ public void externalRef() throws BadLocationException {
 		XMLAssert.testDocumentLinkFor(xml, "src/test/resources/relaxng/main.rng",
 				dl(r(11, 24, 11, 34), "src/test/resources/relaxng/inline.rng"));
 	}
+
+	@Test
+	public void emptyIncludeHref() throws BadLocationException {
+		String xml = "<grammar xmlns=\"http://relaxng.org/ns/structure/1.0\" >\n"
+				+ "	<include href=\"\">\n"
```

Comment:
I wonder if we should have tests with  `<include href=\">` + `<include href=>` +  `<include href>` + `<include>`